packages/core/package.json

```diff
@@ -185,7 +185,8 @@
     "test-unit-db2": "cross-env DIALECT=db2 yarn _test-unit",
     "test-unit-ibmi": "cross-env DIALECT=ibmi yarn _test-unit",
     "test-unit-snowflake": "cross-env DIALECT=snowflake yarn _test-unit",
-    "test-unit-all": "yarn test-unit-mariadb && yarn test-unit-mysql && yarn test-unit-postgres && yarn test-unit-mssql && yarn test-unit-sqlite && yarn test-unit-snowflake && yarn test-unit-db2 && yarn test-unit-ibmi",
+    "test-unit-cockroachdb": "cross-env DIALECT=cockroachdb yarn _test-unit",
+    "test-unit-all": "yarn test-unit-mariadb && yarn test-unit-mysql && yarn test-unit-postgres && yarn test-unit-mssql && yarn test-unit-sqlite && yarn test-unit-snowflake && yarn test-unit-db2 && yarn test-unit-ibmi && yarn test-unit-cockroachdb",
```

Comment:
Quick note; we don't use this script in the CI. It's easier to see which dialect fails if we use them as separate steps, so you need to add it to the following file as well;
https://github.com/divye11/sequelize-cockroachdb/blob/4cc33617ab40c01a8ef3531ae51222d15685315f/.github/workflows/ci.yml#L61-L99

Reply:
Thanks, added those!